docs/dev/setup.md

```diff
@@ -0,0 +1,430 @@
+
+    In general, the installation steps for service catalog are described [here](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install.md).
+
+    Alternatively, the following sequence of commands should work, too.
+
+    ```bash
+    kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+
+    helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+
+    helm search service-catalog
+
+    kubectl create clusterrolebinding tiller-cluster-admin  --clusterrole=cluster-admin --serviceaccount=kube-system:default
+
+    kubectl -n kube-system patch deployment tiller-deploy -p '{"spec": {"template": {"spec": {"automountServiceAccountToken": true}}}}'
+
+    helm install svc-cat/catalog  --name catalog --namespace catalog
+    ```
+
+>**Note:** In order to reuse the docker daemon and speed up local development check the [Reusing the Docker daemon section of the minikube getting started guide](https://kubernetes.io/docs/getting-started-guides/minikube/#reusing-the-docker-daemon)
+
+___
+
+## Setup Service Manager Components
+
+This step shows how to setup the Service Manager as well as the Service Manager Proxies (CF and K8S). There are a couple of ways things can be done.
+For development purposes, the easiest way is to have everything (Service Manager, CF Proxy, K8S Proxy, Dummy Broker) to be running locally on your machine and have them configured to talk to Minikube and PCF Dev. Alternatively, the components can be setup to run on PCF Dev or Minikube and you may run on `localhost` only the component that you are currently working on.
+
+The next steps describe how each component can be setup either to run locally or to run on a locally prepared platform (local CF / K8S minikube)
+
+### Setup the Service Manager
+
+Clone the repository:
+
+```bash
+git clone https://github.com/Peripli/service-manager.git
+
+```
+
+#### Run Service Manager on localhost
+
+>**Prerequisites:**
+>
+>Setup Postgresql
+>
+>In order for the Service Manager to run locally, itrequires a >Postgresql DB.
+>
+> * Install on your machine
+>
+>    Windows users may follow [this guide]>>(http://www.postgresqltutorial.com/install-postgresql/) in order to install postgresql.
+>
+> * Spin up a postgres docker container
+>
+>    ``` bash
+>    docker run --name postgres-docker -d -p 5432:5432 -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres postgres
+>    ```
+>
+>The DB should be accessible on the ip that the docker machine is running on (docker-machine ip) and port 5432.
+
+* Modify the contents of application.yml to point to the configured Postgresql
+
+    ``` yml
+    server:
+      requestTimeout: 3000
+      shutdownTimeout: 3000
+      port: 8080
+    log:
+      level: debug
+      format: text
+    db:
+      name: sm-postgres
+      uri: postgres://postgres:postgres@{postgres-ip}:5432/postgres?sslmode=disable
+    ```

```

Comment:
There is an updated version of application.yml that link like:

```

server:
  requestTimeout: 3000
  shutdownTimeout: 3000
  port: 8080
log:
  level: debug
  format: text
db:
  uri: postgres://postgres:postgres@192.168.99.100:5432/postgres?sslmode=disable
token_issuer_url: https://uaa.local.pcfdev.io

```

I do no know what is the address of the issuer. Need to be clarified. @georgifarashev ?